README.md

```diff
@@ -18,13 +16,15 @@
 
 ## Installation
 
-`installation instructions go here`
+- In order to access this application, all you need to do is click the hyperlink or picture of the application above and you will be directed to the website.
```

Comment:
I like to type out the full url at least once somewhere in the readme. This section is typically where I do it.

```suggestion
- In order to access this application, all you need to do is navigate to <https://yard-pal.herokuapp.com/>, or click the hyperlink or picture of the application above and you will be directed to the website.
```